applications/monitoring/values.yaml

```diff
@@ -24,50 +122,64 @@ chronograf:
       memory: 30Gi
       cpu: 4
 
-  ## Use our own Gafaelfawr ingress.
+  # -- Chronograf ingress
   ingress:
-    ## We will use Gafaelfawr ingresses instead
     enabled: false
+    tls: false
+    hostname: ""
+    className: "nginx"
+    path: /chronograf(/|$)
 
   ## Enable OAuth
+  # -- Enable Chronograf oauth
+  # Never enable it: it breaks the deployment because it's expecting a static
+  # TOKEN_SECRET.  Instead, to get oauth working, leave this setting as-is
+  # and just configure all the correct environment variables (see below).
   oauth:
-    ## This is a lie: see below.
     enabled: false
-  ## OAuth Settings for OAuth Providers
-  ## We do not set these here.  What we do is set:
-  ##  - TOKEN_SECRET
-  ##  - GH_CLIENT_ID
-  ##  - GH_CLIENT_SECRET
-  ##  - GH_ORGS
-  ## in the environment.  The secrets should come from the "monitoring"
-  ## secret, which is a VaultSecret, and the rest can just be injected
-  ## into the deployment env directly.
-
-  ## Extra environment variables that will be passed onto deployment pods
+
+  # -- Environment for chronograf
   env:
+    BASE_PATH: /chronograf
     CUSTOM_AUTO_REFRESH: "1s=1000"
-    GH_CLIENT_ID: ""  # Must be specified for each endpoint for the callback
-    GH_ORGS: "lsst-sqre"
     HOST_PAGE_DISABLED: true
     INFLUXDB_URL: "https://monitoring.lsst.codes"  # Expect this to change
     INFLUXDB_ORG: "square"
-  ## INFLUXDB_TOKEN should be in the monitoring secret as well as
-  ## TOKEN_SECRET and GH_CLIENT_SECRET.  Note that INFLUX_TOKEN is for
-  ## InfluxDBv1 and INFLUXDB_TOKEN is for v2.
+
+  # -- Chronograf will read environment variables from this secret
+  # It expects keys generic_client_id, generic_client_secret,
+  # and token_secret (assuming you want generic OIDC auth talking to your
+  # local Gafaelfawr, which is usually the case).  You should also have
+  # INFLUXDB_TOKEN for InfluxDBv2.  Check the Chronograf docs if you are
+  # using some other setup (e.g. InfluxDBv1 for a database, or GitHub for
+  # an auth provider.)
```

Comment:
This is a really long comment that I think is duplicating the documentation in `secrets.yaml`. Does someone writing a values file for a specific environment need to know about the construction of the secret? I think this is all done by the `VaultSecret` and `secrets.yaml`, isn't it? In which case it's probably best not to put this into the user-facing documentation. (Also, `secrets.yaml` has all of those keys in all-uppercase, but here they're in all-lowercase. If they're really supposed to be in lowercase, `secrets.yaml` should match.)